backend/clickhouse/migrations/000135_create_readonly_user.up.sql

```diff
@@ -0,0 +1,25 @@
+CREATE USER IF NOT EXISTS highlight_readonly IDENTIFIED BY '' SETTINGS readonly = 2;
+CREATE ROLE IF NOT EXISTS highlight_readonly_role;
+GRANT SELECT ON error_groups TO highlight_readonly_role;
+GRANT SELECT ON error_objects TO highlight_readonly_role;
+GRANT SELECT ON errors_joined_vw TO highlight_readonly_role;
+GRANT SELECT ON sessions TO highlight_readonly_role;
+GRANT SELECT ON sessions_joined_vw TO highlight_readonly_role;
+GRANT SELECT ON session_events_vw TO highlight_readonly_role;
+GRANT SELECT ON fields TO highlight_readonly_role;
+GRANT SELECT ON traces TO highlight_readonly_role;
+GRANT SELECT ON traces_sampling_new TO highlight_readonly_role;
+GRANT SELECT ON logs TO highlight_readonly_role;
+GRANT SELECT ON logs_sampling TO highlight_readonly_role;
+CREATE ROW POLICY IF NOT EXISTS error_groups_readonly ON error_groups USING ProjectID = getSetting('SQL_highlight_project_id') TO highlight_readonly_role;
```

Comment:
do these not have to be `AS RESTRICTIVE` to limit access?

Reply:
`PERMISSIVE` / `RESTRICTIVE` are to detemine how policies interact with other policies - in this case only one policy applies per table so there's no difference